### PR TITLE
fix #722:Add beneficiary fab button starts activity

### LINKED ISSUE #722: Clicking on Add beneficiary fab button should open new activity  instead of fragment replacement
**Summary:**

As suggested by @therajanmaurya  in the PR https://github.com/openMF/mifos-mobile/pull/705 comments that clicking the fab button while adding beneficiary should open a new activity instead of fragment replace.

I am working on it.

### PR DESCRIPTION
**Pull request by AkshGautam**
Fixes #722 

@therajanmaurya @dilpreet96  please review. As far as I understood from your suggestion I have implemented the same. If you need any amends. I would be happy to make them.
Thanks
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -100,35 +97,34 @@
...
-                <action android:name="com.google.android.gms.iid.InstanceID"/>
+                <action android:name="com.google.android.gms.iid.InstanceID" />
             </intent-filter>
         </service>
-
         <service
             android:name=".utils.gcm.RegistrationIntentService"
-            android:exported="false">
-        </service>
+            android:exported="false"></service>
```
Comment (therajanmaurya) [inline app/src/main/AndroidManifest.xml:124]: Revert back all the formatting change that does not belong to your change.

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by therajanmaurya** [comment]
Fix Conflicts.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (AkshGautam) [reply app/src/main/AndroidManifest.xml:124]: @therajanmaurya  sure sir I would do that.

**Comment by therajanmaurya** [comment]
Close with #856

**Pull request closed without merging**